backend/tests/v5/unit/middleware/dataConverter/outputs/teamspaces/projects/models/commons/tickets.test.js

```diff
@@ -440,9 +442,122 @@ const testSerialiseTicketList = () => {
 	});
 };
 
+const testSerialiseTicketHistory = () => {
+	describe('Serialise ticket history', () => {
+		const history = times(5, (i) => {
+			if (i % 2 === 0) {
+				return ({
+					author: generateRandomString(),
+					changes: {
+						[generateRandomString()]: {
+							from: null,
+							to: new Date(),
+						},
+					},
+					timestamp: generateRandomDate(),
+				});
+			}
+			if (i % 3 === 0) {
+				return ({
+					author: generateRandomString(),
+					changes: {
+						[generateRandomString()]: {
+							from: null,
+							to: generateUUID(),
+						},
+					},
+					timestamp: generateRandomDate(),
+				});
+			}
+			return ({
+				author: generateRandomString(),
+				changes: {
+					[generateRandomString()]: {
+						from: generateRandomString(),
+						to: generateRandomString(),
+					},
+				},
+				timestamp: generateRandomDate(),
+			});
+		});
+
+		test('Should respond with the correct format', () => {
```

Comment:
I think it would be the best to follow this example

<img width="1001" height="523" alt="Image" src="https://github.com/user-attachments/assets/047396e8-e4e7-471f-abff-dd99550f915d" />

have test cases where you provide the history before and after i.e. the expected outcome. So in this case you can provide one test case for each scenario: property date/uuid conversion, module date/uuid conversion, import date/uuid conversion. Also it will be easier to extend if we decide we wanna convert some additional properties in the future